spec/spec_helper.rb

```diff
@@ -93,4 +93,7 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 =end
+  config.before(:suite) do
+    Rails.application.load_seed
```

Comment:
I don't quite agree with loading the seeds before running the tests. At least not because of the way we are using the seeds, which is in part to populate our local DBs with some data to test our local environment.
If the seeds were being used to populate the production DB in a clean and running state, other would be the case.

Reply:
Never mind, I just realized that the extra data is being created only in development, not test. Sorry for the confusion!

Reply:
Np!